hbase-client/src/main/java/org/apache/hadoop/hbase/filter/MultiRowRangeFilter.java

```diff
@@ -71,6 +72,22 @@ public MultiRowRangeFilter(List<RowRange> list) {
     this.ranges = new RangeIteration(rangeList);
   }
 
+  /**
+   * @param rowKeyPrefixes the array of byte array
```

Comment:
I think we need a careful javadoc here, say why we need the public method ... because exposing an MultiRowRangeFilter constructor with rowKeyPrefixes  looks very strange if no doc here.

Reply:
OK, I added explanation for this constructor.